webform_import.module

```diff
@@ -407,6 +419,10 @@ function _webform_import_import($form, $form_state, $file) {
             $sid = intval($cell_value); // Valid input security (integer values only).
           }
         }
+        // serial number for submission
```

Comment:
Should start with a capital `S`, and preferably also reformatted as a sentence. :)